scripts/zones/Pashhow_Marshlands/mobs/NiZho_Bladebender.lua

```diff
@@ -7,6 +7,13 @@ require("scripts/globals/hunts")
 -----------------------------------
 local entity = {}
 
+entity.onMobWeaponSkillPrepare = function(mob, target)
+    -- Spams headbutt below 30% HP
```

Comment:
Does that mean below 30 he won't use anything else?

Reply:
This is just my interpretation based off of wiki's note of: "Seems to favor the Quadav move [Head Butt](https://ffxiclopedia.fandom.com/wiki/Head_Butt) and will spam it at 30%."

I don't have any video evidence of this, so I can remove this logic if needed.

Reply:
I think that type of thing is best to hold off till we implement a way to prioritize skill selections (and hopefully in a better manner than we do spell types. So many mobmods :weary: )

Reply:
Removed the logic from its lua